nilearn/externals/skimage/peak.py

```diff
@@ -0,0 +1,161 @@
+import numpy as np
+import scipy.ndimage as ndi
+from _rank_order import rank_order
```

Comment:
Fix it, hopefully that should get Travis green and I'll take a closer look at your PR.